src/containers/Detail/components/TableDetail/styles.module.scss

```diff
@@ -0,0 +1,10 @@
+@import '@scss/mixins';
+
+.tableContainer {
+  border: 1px solid rgba(108, 117, 125, 25%) !important;
+}
+
+th,
+td :last-child {
```

Comment:
Do not use default selector from html. Using class name instead. This will help avoid conflicting to another module.

Reply:
clear done